functests/performance/performance.go

```diff
@@ -92,13 +92,13 @@ var _ = Describe("performance", func() {
 
 		// Check /usr/local/bin/pre-boot-tuning.sh existence under worker's rootfs
 		const perfRtKernelPrebootTuningScript = "/usr/local/bin/pre-boot-tuning.sh"
-		It(perfRtKernelPrebootTuningScript+" should exist on the nodes", func() {
+		It(perfRtKernelPrebootTuningScript+"[test_id:28523][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] should exist on the nodes", func() {
```

Comment:
hm. may be shoould be this?
		It("[test_id:28523][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] "+perfRtKernelPrebootTuningScript+" should exist on the nodes", func() {

Reply:
hm, IMHO even better to use `fmt.Sprintf()`